src/overlays/actors/ovl_Boss_Fd/z_boss_fd.c

```diff
@@ -21,51 +105,1974 @@ const ActorInit Boss_Fd_InitVars = {
+                spB8.x = temp_X + this->holePosition.x;
+                spB8.y = 100.0f;
+                spB8.z = temp_Z + this->holePosition.z;
+                func_8002836C(globalCtx, &spB8, &spD0, &spC4, &D_808D1A58, &D_808D1A5C, 500, 0xA, 20);
+            }
+        }
+
+        for (i = 0; i < 8; i++) {
+            spD0.x = Math_Rand_CenteredFloat(20.0f);
+            spD0.y = Math_Rand_ZeroFloat(10.0f);
+            spD0.z = Math_Rand_CenteredFloat(20.0f);
+            spC4.y = 0.4f;
+            spC4.x = Math_Rand_CenteredFloat(0.5f);
+            spC4.z = Math_Rand_CenteredFloat(0.5f);
+            spB8.x = Math_Rand_CenteredFloat(60.0) + this->holePosition.x;
+            spB8.y = Math_Rand_ZeroFloat(40.0f) + 100.0f;
+            spB8.z = Math_Rand_CenteredFloat(60.0) + this->holePosition.z;
+            func_808CADC0(this->particles, &spB8, &spD0, &spC4, (s16)Math_Rand_ZeroFloat(1.5f) + 6);
+        }
+    }
+
+    if ((this->fireBreathTimer != 0) && (this->fireBreathTimer < 0x11)) {
+        sp106 = (this->fireBreathTimer >= 6) ? 0xFF : this->fireBreathTimer * 0x32;
+    }
+    if (sp106 != 0) {
+        f32 spB0;
+        f32 spAC;
+        Vec3f spA0 = { 0.0f, 0.0f, 0.0f };
+        Vec3f sp94;
+        Vec3f sp88 = { 0.0f, 0.0f, 0.0f };
+        Vec3f sp7C;
+        this->fogMode = 2;
+        spA0.z = 30.0f;
+
+        Audio_PlaySoundGeneral(NA_SE_EN_VALVAISA_FIRE - SFX_FLAG, &D_808D1A28, 4, &D_801333E0, &D_801333E0,
+                               &D_801333E8);
+        sp7C = this->fireBreathSpawnPoint;
+        spAC = (this->actor.posRot.rot.y / 32768.0f) * M_PI;
+        spB0 = (((-this->actor.posRot.rot.x) / 32768.0f) * M_PI) + 0.3f;
+        Matrix_RotateY(spAC, 0);
+        Matrix_RotateX(spB0, 1);
+        Matrix_MultVec3f(&spA0, &sp94);
+        func_808CAFF4(this->particles, &sp7C, &sp94, &sp88, 50.0f * Math_Sins(this->varianceTimer * 0x2000) + 300.0f,
+                      sp106, this->actor.posRot.rot.y);
+        sp7C.x += sp94.x * 0.5f;
+        sp7C.y += sp94.y * 0.5f;
+        sp7C.z += sp94.z * 0.5f;
+        func_808CAFF4(this->particles, &sp7C, &sp94, &sp88, 50.0f * Math_Sins(this->varianceTimer * 0x2000) + 300.0f,
+                      sp106, this->actor.posRot.rot.y);
+        spA0.x = 0.0f;
+        spA0.z = 0.0f;
+        spA0.y = 17.0f;
+
+        for (i = 0; i < 6; i++) {
+            spAC = Math_Rand_ZeroFloat(6.2831855f);
+            spB0 = Math_Rand_ZeroFloat(6.2831855f);
+            Matrix_RotateY(spAC, 0);
+            Matrix_RotateX(spB0, 1);
+            Matrix_MultVec3f(&spA0, &sp94);
+            sp88.x = (sp94.x * -10) / 100;
+            sp88.y = (sp94.y * -10) / 100;
+            sp88.z = (sp94.z * -10) / 100;
+            func_808CADC0(this->particles, &this->fireBreathSpawnPoint, &sp94, &sp88,
+                          (s16)Math_Rand_ZeroFloat(2.0f) + 8);
+        }
+    }
+
+    if ((this->actor.posRot.pos.y < 90.0f) || (700.0f < this->actor.posRot.pos.y) ||
```

Comment:
I prefer constants on the right hand side and I would assume the original code has them on the right, but are we doing that?

Reply:
I normally prefer the constants on the right, but joint inequalities that act like intervals (such as the one above) I prefer the inequalities to all go the same direction.